src/quemb/molbe/solver.py

```diff
@@ -35,6 +35,9 @@
 from quemb.shared.helper import delete_multiple_files, unused
 from quemb.shared.manage_scratch import WorkDir
 
+Solvers: TypeAlias = Literal["CCSD", "MP2", "CCSD", "FCI", "HCI", "SHCI", "SCI", "DMRG"]
```

Comment:
@ShaunWeatherly 
Do you want to have all possible DMRG literals here or is `"DMRG"` enough.

Reply:
resolved by chatting in person